landlab/components/priority_flood_flow_router/priority_flood_flow_router.py

```diff
@@ -536,6 +517,20 @@ def __init__(
         # Create properties specific to RichDEM
         self._create_richdem_properties()
 
+    @staticmethod
+    def load_richdem():
+        try:
+            import _richdem  # noqa: F401
+            import richdem
+        except ModuleNotFoundError as exc:
+            raise ModuleNotFoundError(
+                "PriorityFloodFlowRouter requires richdem but richdem is not installed."
+                "You can install richdem either from source, "
+                "(https://github.com/r-barnes/richdem), or through conda "
+                "(conda install richdem -c conda-forge) or pip (pip install richdem)."
+            ) from exc
```

Comment:
@BCampforts Here is the error message that a user would see if they try to run `PriorityFloodFlowRouter` with *richdem*.

Reply:
That looks better indeed!